services/web-app/components/storybook-demo/storybook-demo.module.scss

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+@use '@carbon/react/scss/spacing' as spacing;
+@use '@carbon/react/scss/theme' as theme;
+@use '@carbon/react/scss/utilities/convert' as convert;
+
+// override carbon, label left padding
+:global(.cds--label) {
+  padding-left: spacing.$spacing-05;
+}
+
+:global(.cds--dropdown) {
+  border-bottom: 1px solid theme.$layer-selected-01;
+}
+
+.theme-selector {
+  :global(.cds--dropdown) {
+    border-right: 1px solid theme.$layer-selected;
```

Comment:
Instead of a border I would use the :after pseudo element with a width of 1px to fill the space that is already there.